webcompat/templates/dashboard/triage/top-bar.html

```diff
@@ -0,0 +1,11 @@
+<style>
+  .wc-TopBar {
+    font-size: 1rem;
+    background-color: #ffc900;
+    padding: 1em;
+  }
+</style>
+
+<header class="wc-TopBar" role="banner">
+  <h1 style="margin: 0; font-size: 1.3em">Triage Dashboard</h1>
```

Comment:
We break one test with this change. This is fixable. 
Either by modifying the test by adding the style attribute 
OR by pushing the style into the `style` element with a `.wc-TopBar h1 {margin: 0; font-size: 1.3em} rule.`

```
→ nosetests
................................................F......................
======================================================================
FAIL: Request to /dashboard/triage should be 200.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/karl/code/webcompat.com/tests/test_urls.py", line 139, in test_dashboard_triage
    self.assertTrue('<h1>Triage Dashboard</h1>' in rv.data)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 71 tests in 2.823s

FAILED (failures=1)
```